src/evergreen/cli/main.py

```diff
@@ -371,6 +371,34 @@ def build_stats(ctx, build_id, tasks):
         click.echo(fmt_output(fmt, build.get_metrics().as_dict(include_children=tasks)))
 
 
+@cli.command()
+@click.pass_context
+@click.option("--project", required=True)
```

Comment:
Could you add `help` parameters to these options?

Reply:
Done.